### PR TITLE
[PRO-320] Finish up resources card

### PR DESCRIPTION
**Pull request by eddie-m-m**
<h2>Changes</h2>

- Adds comments count to resource card
- Adds comments icon to resource card, linking to resources details (`ResourceView`)
- Ensures comments count in `ResourceView` is updated immediately after a comment is created

<h2>Testing</h2>

**NOTE**: Depends on PRO-324 which must be merged first

1. Navigate to Resources
2. Click on comment icon on any resource card
3. Add comments to your heart's delight
4. Observe updated comments count both within the resource details page and after navigating back to Resources list.

### CODE REVIEW

**Comment by eddie-m-m** [comment]
@tasandberg It looks like something is breaking with the `tolgee-pull` script. I'm noticing now I can't build locally either.

File: src/components/Resources/ResourceCard.tsx [hunk 1]
```diff
@@ -138,23 +139,34 @@ export default function ResourceCard({ resource, onUpdate }: IResourceCard) {
...
+              } else {
+                showUnauthorizedToast()
+              }
+            }}
+          />
+          <Link
+            to={`/resources/${resource.id}`}
+            className="text-decoration-none"
+          >
+            <div className={'d-flex flex-row gap-2'}>
```
Comment (tasandberg) [inline src/components/Resources/ResourceCard.tsx:164]: To match the voting control styles and keep everything consistant: 
```suggestion
            <div className={'d-flex flex-row gap-1 text-dark'}>
```

**Review by tasandberg — COMMENTED** [review]
Nicely done, small style nit.

**Comment by eddie-m-m** [comment]
I incorporated the style adjustment. 

I had started with `gap-1` but it seemed that the distance (visually) between the count and the comments icon was smaller than that between the thumbs icons and their numbers.

**Review by tasandberg — APPROVED** [review]
Nice work! Looks great on my end.

**Pull request merged**